Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -144,17 +152,40 @@ class PlaceDetailsCardView: UIView {
         layer.shadowOpacity = 0.4
     }
 
+    var stackViewTopToEventViewBottomConstraint: NSLayoutConstraint!
+    var stackViewTopToContentViewTopConstraint: NSLayoutConstraint!
+
     private func setupViews() {
-        backgroundColor = Colors.detailsViewCardBackground
+        backgroundColor = Colors.detailsViewCardBackground // cannot be transparent to display shadow
+        contentView.backgroundColor = Colors.detailsViewCardBackground
+
         layer.cornerRadius = 10
+        contentView.layer.cornerRadius = 10
+        contentView.layer.masksToBounds = true
+
+        addSubview(contentView)
```

Comment:
Is there any reason why you didn't just insert the new view into the stack view at the very top?